### PR TITLE
Added and implemeted feature 102 LD Working with context files

### PR DESCRIPTION
**Pull request by jicarretero**

### CODE REVIEW

File: features/102.ngsild.working_with_context.feature [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  Scenario: Checking the Service Health
+    When I prepare a GET HTTP request to "http://localhost:1026/version"
+    And   I perform the query request
+    Then  I receive a HTTP "200" response code
+
+  # Request 2: Create data entities
+  Scenario: Create a new data entity
+    When  I prepare a POST HTTP request to "http://localhost:1026/ngsi-ld/v1/entities/"
+    And   I set header Content-type to application/ld+json
+    And   I set the body request as described in 102.ld.02.request.json
```
Comment (flopezag) [inline features/102.ngsild.working_with_context.feature:35]: Please, use " for the file name to follow the style

File: features/102.ngsild.working_with_context.feature [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    And   I perform the request
+    Then  I receive a HTTP "201" response code
+    And   I have the header "Location" with value /ngsi-ld/v1/entities/urn:ngsi-ld:Building:farm001
+
+  # Request 3: New data enities
+  Scenario: Each subsequent entity must have a unique id for the given type
+    When  I prepare a POST HTTP request to "http://localhost:1026/ngsi-ld/v1/entities/"
+    And   I set header Content-type to application/json
+    And   I set header Link to <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
+    And   I set the body request as described in 102.ld.03.request.json
```
Comment (flopezag) [inline features/102.ngsild.working_with_context.feature:45]: Idem

File: features/data/102.ld.working_with_context/response102-04.excludes [hunk 3]
```diff
@@ -0,0 +1,2 @@
+root[0]['@context']
+root[1]['@context']
```
Comment (flopezag) [inline features/data/102.ld.working_with_context/response102-04.excludes]: Why @context is an exclusion?

File: features/steps/102.ngsild.working_with_context.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import json
+
+__logger__ = getLogger(__name__)
+
+
+@given(u'I set the tutorial 102 NGSI-LD')
+def step_impl(context):
+    context.data_home = join(join(join(CODE_HOME, "features"), "data"), "102.ld.working_with_context")
+
+@step(u'I have the header "{header}" with value {value}')
```
Comment (flopezag) [inline features/steps/102.ngsild.working_with_context.py]: I prefer this definition:

I have the header "{header}" with value "{value}"

**Review by flopezag — CHANGES_REQUESTED** [review]

File: features/102.ngsild.working_with_context.feature (hunk 1 above)
Comment (flopezag) [reply features/102.ngsild.working_with_context.feature:35]: A technical debt (Issue #34) was created to tackle with this issue in the next releases.

NTC

File: features/102.ngsild.working_with_context.feature (hunk 2 above)
Comment (flopezag) [reply features/102.ngsild.working_with_context.feature:45]: A technical debt (Issue #34) was created to tackle with this issue in the next releases.

NTC

File: features/data/102.ld.working_with_context/response102-04.excludes (hunk 3 above)
Comment (jicarretero) [reply features/data/102.ld.working_with_context/response102-04.excludes]: @context differs between tutorial en the one produced by Context Broker -- It is reported as a discrepancy between the tutorial and the Broker output.

File: features/steps/102.ngsild.working_with_context.py (hunk 4 above)
Comment (jicarretero) [reply features/steps/102.ngsild.working_with_context.py]: Ok.

Comment (flopezag) [reply features/steps/102.ngsild.working_with_context.py]: LGTM

File: features/data/102.ld.working_with_context/response102-04.excludes (hunk 3 above)
Comment (flopezag) [reply features/data/102.ld.working_with_context/response102-04.excludes]: LGTM

**Review by flopezag — APPROVED** [review]

**Pull request merged**